### PR TITLE
Implement /friends/suggested route [PAR-77]

### PR DESCRIPTION
**Pull request by Hashiphoto**
Figured it out! And they say it couldn't be done.

### CODE REVIEW

File: db.js [hunk 1]
```diff
@@ -295,36 +295,27 @@ var friends = (function () {
...
+              "AND b.status = 'friends' " +
+              "AND NOT EXISTS (SELECT * FROM friendships WHERE user_id = a.user_id AND friend_id = u.id) " +
+              "LIMIT 10",
+            [currentUserId]
+          )
+        )
+        .then(([rows, fields]) => {
+          return rows;
+        });
     },
```
Comment (vastImmortalSuns) [inline db.js:319]: Congrats on the single query!

**Comment by vastImmortalSuns** [comment]
I'm more than fine with merging this. Just out of curiosity, why did you settle on 10 friends being returned specifically?

**Comment by Hashiphoto** [comment]
@vastImmortalSuns I just thought it'd be a nice amount. Looking at the page now, I think having more than that would be overwhelming. I want to keep it fast

**Comment by vastImmortalSuns** [comment]
Gotcha. Well that sounds like a merge to me. Nice work figuring out the single query problem!

**Pull request merged**